src/sentry/runner/initializer.py

```diff
@@ -187,6 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    if lvl in logging._levelNames:
```

Comment:
I think this would get blocked by `click` at the runner anyways, so you wouldn't be able to set an invalid value.